src/frontend/src/pages/CreateChangeRequestPage/CreateChangeRequestView.tsx

```diff
@@ -276,20 +308,13 @@ const CreateChangeRequestsView: React.FC<CreateChangeRequestViewProps> = ({
               </Button>
             </Grid>
           </Grid>
-        </PageBlock>
-        <PageBlock title="Proposed Solutions">
-          <CreateProposedSolutionsList proposedSolutions={proposedSolutions} setProposedSolutions={setProposedSolutions} />
-        </PageBlock>
-        <Box textAlign="right">
-          <NERFailButton variant="contained" onClick={handleCancel} sx={{ mx: 1 }}>
-            Cancel
-          </NERFailButton>
-          <NERSuccessButton variant="contained" type="submit" sx={{ mx: 1 }}>
-            Submit
-          </NERSuccessButton>
-        </Box>
-      </form>
-    </PageLayout>
+          <Grid item xs={1} />
```

Comment:
nice job keeping to the mocks, but I think we can use some more of the screen space. I'd say remove this grid item and change the top one to be 6 instead of 5